lib/gems/pending/util/miq_object_storage.rb

```diff
@@ -1,12 +1,16 @@
+require 'net/protocol'
 require 'util/miq_file_storage'
 
 class MiqObjectStorage < MiqFileStorage::Interface
   require 'util/object_storage/miq_s3_storage'
   require 'util/object_storage/miq_ftp_storage'
+  require 'util/object_storage/miq_swift_storage'
 
   attr_accessor :settings
   attr_writer   :logger
 
+  DEFAULT_CHUNKSIZE = Net::BufferedIO::BUFSIZE
```

Comment:
For some reference, this comes from `net/protocol`, and it is the same chunk size that is used by `Net::FTP` when uploading.  Figured it was a reasonable value to pick out of a hat and use here.